mapper/mapper_test.go

```diff
@@ -1,10 +1,8 @@
 package mapper
 
 import (
```

Comment:
Please organize the imports the following way https://github.com/golang/go/wiki/CodeReviewComments#imports.